cogs/cmd_interface/leaderboard.py

```diff
@@ -3,20 +3,32 @@
 from discord.ext import commands
 from lib.embed.interactable_leaderboard_embed import RankingView, InteractableLeaderboardEmbed
 from utils.llc_datetime import LLCMonth
+from lib.embed.leaderboard_embed import role_emojies
```

Comment:
Refactor role_emojies ra một file riêng đi.

@cdf144 I'm pinging you to instruct him on ideas of best practices.

Reply:
_nervous sweating_

Yea I think refactoring the dict onto a separate file allows for better extensibility and reduces coupling (dependency on `leaderboard_embed`). `role_emojies` wasn't on a separate file because it was only used by one module, but now we should do it.